great-idea/index.html

```diff
@@ -15,9 +15,102 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <!-- Main Cointainer-->
+    <div id="main_container">
+  <!--Navigation Container-->
+  <div id="nav">
+    <nav>
+        <a href="services.html">Services</a> 
+        <a href="/css/">Product</a> 
+        <a href="vision.html">Vision</a> 
+        <a href="/jquery/">Features</a>
+        <a href="/html/">About</a> 
+        <a href="/css/">Contact</a>
+    </nav>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </div>
+  <!--Heading Container-->
+  <div id="heading">
+    <!--Heading Text Container-->
+    <div id="heading_text">
```

Comment:
As a best practice, I would recommend trying to add at least one line break of white space in between each element. It just makes things more readable and easier for you to quickly scan later on.